lib/redux/actions/async.js

```diff
@@ -236,6 +236,7 @@ export function doDeviceUpload(driverId, opts, utc) {
     const version = versionInfo.semver;
     const { devices, os, targetTimezones, uploadTargetUser } = getState();
     const targetDevice = _.findWhere(devices, {source: {driverId: driverId}});
+    opts = opts || {};
```

Comment:
Ha, I totally forgot that I'm allowed to use ES6 in that file, even though there are ES6 `const`s everywhere! :)